mesh.proto

```diff
@@ -1166,6 +1166,19 @@ message LogRecord {
   Level level = 4;
 }
 
+message QueueStatus {
+  /* Last attempt to queue status */
+  uint32 res = 1;
+  /* Free entries in the outgoing queue */
+  uint32 free = 2;
```

Comment:
at least free and maxlen are 'unsigned int' in the code, no need to have an uint32 here. Can you add an 'option' to limit them to uint8?